### ISSUE TITLE
deploy failed due to "can't delete bridge br-b-vm-t1: Operation not permitted"

### ISSUE THREAD

**Issue by stephenxs** [Azure/sonic-mgmt#1375]
**Description**
Deploy failed due to "can't delete bridge br-b-vm-t1: Operation not permitted" when add/remove topo.
With the PR [move route template from eos to ptf exabgp #1233](https://github.com/azure/sonic-mgmt/pull/1233) reverted the issue won't be reproducible. 
But I'm not sure whether it is caused by the commit.

**Steps to reproduce the issue:**
1. deploy the testbed in t0.
2.
3.

**Describe the results you received:**
```
TASK [vm_set : Unbind topology t0 to VMs. base vm = VM0601] ******************** 

Saturday 08 February 2020  13:52:08 +0000 (0:00:00.059)       0:00:38.576 ***** 

fatal: [ARC-HOST121]: FAILED! => {"changed": false, "msg": "ret_code=1, error message=can't delete bridge br-b-vm-t1: Operation not permitted\n. cmd=brctl delbr br-b-vm-t1"} 

```

**Describe the results you expected:**
No error should be observed.

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by lguohan** [comment]
we need to use the cleanup script to clean up the old testbed.

**Comment by Blueve** [comment]
I am not sure is there a command to run this script? But I tried following command and it works:
```bash
~/sonic-mgmt/ansible/roles/vm_set/templates$ sudo bash ./cleanup.sh.j2
```
Please aware that this command will remove all testbed related creator including sonic-mgmt container.

**Comment by stephenxs** [comment]
resolved after updating server to Ubuntu 18.04
